mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -2586,6 +2586,1276 @@
+													"    pm.response.to.have.status(400);",
+													"});",
+													"",
+													"//Validate response against json api schema",
+													"pm.test(\"Validate schema\", function () {",
+													"     tv4.addSchema(\"JSONAPI.json\", JSON.parse(pm.variables.get(\"schema_jsonapi_content\")));",
+													"    pm.expect(tv4.validate(response, JSON.parse(pm.environment.get(\"schema_jsonapi_content\")))).to.equal(true, \"Schema validation error: \" + JSON.stringify(tv4.error));",
+													"    pm.expect(tv4.missing.length).to.equal(0, \"Missing schemas: \" + JSON.stringify(tv4.missing));",
+													"});",
+													"//Ensure that errors array is not empty",
+													"if(response.errors) {",
+													"    pm.test('Ensure that errors array is not empty', function() {",
+													"        pm.expect(response.errors).to.be.an('array').that.is.not.empty;",
+													"    });",
+													"",
+													"    pm.test('Ensure that response contains expected error message', function(){",
+													"       pm.expect(response.errors[0].title).to.eq('Search parameter cannot be empty'); ",
+													"    });",
+													"}",
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"method": "GET",
+										"header": [
+											{
+												"key": "x-okapi-tenant",
+												"value": "{{xokapitenant}}"
+											},
+											{
+												"key": "x-okapi-token",
+												"value": "{{xokapitoken}}"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": ""
+										},
+										"url": {
+											"raw": "{{protocol}}://{{url}}:{{okapiport}}/eholdings/providers/19/packages?q=",
+											"protocol": "{{protocol}}",
+											"host": [
+												"{{url}}"
+											],
+											"port": "{{okapiport}}",
+											"path": [
+												"eholdings",
+												"providers",
+												"19",
+												"packages"
+											],
+											"query": [
+												{
+													"key": "q",
+													"value": ""
+												}
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "invalid page offset",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"id": "4107bfa1-7d75-4212-a1c6-7f084d0d31de",
+												"exec": [
+													"//Ensure that status is 400",
+													"pm.test(\"Status is 400\", function () {",
+													"    pm.response.to.have.status(400);",
+													"});",
+													"",
+													"//Ensure that response contains expected error message",
+													"pm.test(\"Response must have body with error message\", function () {",
+													"    pm.expect(pm.response.text()).to.be.eq('For input string: \"abc\"');",
+													"});",
```

Comment:
This error response doesn't seem to be correct here -- in mod-kb-ebsco we see an error such as 

```{
    "errors": [
        {
            "title": "The value provided for 'offset' is invalid"
        },
        {
            "title": "Parameter Offset is missing."
        }
    ],
    "jsonapi": {
        "version": "1.0"
    }
}```


Other than this (and this would be more of an issue or something we should look into in the code itself) -- looks good @ivan-brovenko

Reply:
These errors are different because "page" has default, min, max value and the validation occurs before the controller

![page](https://user-images.githubusercontent.com/44401279/49652189-01338180-fa3a-11e8-9e22-9a4f4297049c.PNG)